models/sipnet/R/split_inputs.SIPNET.R

```diff
@@ -30,10 +30,12 @@ split_inputs.SIPNET <- function(settings, start.time, stop.time, inputs, overwri
   names(file) <- paste(start.time, "-", stop.time)
   
   #Changing the name of the files, so it would contain the name of the hour as well.
+  formatted_start <- gsub(' ',"_", as.character(start.time))
+  formatted_stop <- gsub(' ',"_", as.character(stop.time))
   file <- paste0(outpath, "/", prefix, ".",
-                 paste0(start.time%>% as.character() %>% gsub(' ',"_",.),
+                 paste0(formatted_start,
                         "-",
```

Comment:
Nice change! I think this makes the inner paste0 redundant too:
```suggestion
                 formatted_start, "-", formatted_stop, ".clim")
```

Reply:
correct !! , i have already removed it to check the redundancy thing ! i will push the changes